### PR TITLE
Show Keywords on annotation mode sidebar

### LINKED ISSUE #151: [UI] Show keywords in annotation mode
Keywords are quite useful on annotation phase.

### PR DESCRIPTION
**Pull request by leiyre**
fix #151

### CODE REVIEW

File: frontend/components/commons/SideBar.vue [hunk 1]
```diff
@@ -45,6 +45,18 @@
...
+            <div
+              v-for="(counter, label) in getKeywords"
+              :key="label"
+            >
+              <div v-if="counter > 0" class="info">
+                <label>{{ label }}</label>
+                <span class="records-number">{{ counter }}</span>
+              </div>
+            </div>
+          </div>
```
Comment (frascuchon) [inline frontend/components/commons/SideBar.vue]:
```suggestion
              v-for="(counter, keyword) in getKeywords"
              :key="keyword"
            >
              <div v-if="counter > 0" class="info">
                <label>{{ keyword }}</label>
                <span class="records-number">{{ counter }}</span>
              </div>
            </div>
          </div>
```

**Comment by frascuchon** [comment]
@leiyre where is keyword logic for exploration views?

**Comment by leiyre** [comment]
@frascuchon it's the same login for both (annotation and exploration)

**Comment by frascuchon** [comment]
> @frascuchon it's the same login for both (annotation and exploration)

So, annotationProgress contains always any kind of information? I see in the code that the whole side bar content is shown only when there is some annotation progress....

```html
<div v-if="annotationsProgress" v-show="visible" class="sidebar__wrapper">
...
```

It's a bit weird if annotations progress info should be checked only when annotation mode is enabled.

Regarding the annotation enabled conditions, could we change the multi check:

```html
          <div v-if="annotationIsEnabled" class="info">
            <label>All</label>
            <span class="records-number">
              <strong>{{ total }}</strong>
            </span>
          </div>
          <div v-if="annotationIsEnabled" class="info">
            <label>Validated</label>
            ...
          </div>
          <div v-if="annotationIsEnabled" class="info">
            ...
          </div>
          <div v-if="annotationIsEnabled" class="labels">
            ...
          </div>
          <div class="labels">
            <strong v-if="annotationIsEnabled">Keywords:</strong>
            <div
              v-for="(counter, keyword) in getKeywords"
              :key="keyword"
            >
              ...
```

with a common and transparent div:

```html
<div v-if="annotationIsEnabled">
   <div class="info">
   <div class="labels">
....
</div>
```

??

**Comment by leiyre** [comment]
@frascuchon I think we can close this PR, this issue is resolved in #179

**Comment by frascuchon** [comment]
Ok, I will close this PR

**Pull request closed without merging**